conseil-api/src/main/scala/tech/cryptonomic/conseil/api/routes/platform/Api.scala

```diff
@@ -0,0 +1,61 @@
+package tech.cryptonomic.conseil.api.routes.platform
+
+import akka.actor.ActorSystem
+import cats.effect.{ContextShift, IO}
+import com.typesafe.scalalogging.LazyLogging
+import tech.cryptonomic.conseil.api.config.ConseilConfiguration
+import tech.cryptonomic.conseil.api.metadata.{AttributeValuesCacheConfiguration, MetadataService}
+import tech.cryptonomic.conseil.api.routes.platform.data.ApiDataEndpoints
+import tech.cryptonomic.conseil.api.routes.platform.data.tezos.{TezosDataEndpoint, TezosDataOperations}
+import tech.cryptonomic.conseil.api.routes.platform.discovery.tezos.TezosPlatformDiscoveryOperations
+import tech.cryptonomic.conseil.common.cache.MetadataCaching
+import tech.cryptonomic.conseil.common.config.MetadataConfiguration
+import tech.cryptonomic.conseil.common.generic.chain.PlatformDiscoveryOperations
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success}
+
+trait Api {
+  def dataEndpoint(metadataService: MetadataService): ApiDataEndpoints
+  def discoveryOperations: PlatformDiscoveryOperations
+}
+
+class TezosApi(metadataOverrides: MetadataConfiguration, server: ConseilConfiguration)(
```

Comment:
Why isn't this splitted to a separate file, like the data endpoints?

Reply:
No particular reason, besides the fact that endpoints are in specific packages and I couldn't find better place for it. I extracted that to `TezosApi` file.